systests/rs-security/src/test/java/org/apache/cxf/systest/jaxrs/security/oidc/filters/OIDCFiltersTest.java

```diff
@@ -111,45 +101,6 @@ public void testClientCodeRequestFilter() throws Exception {
         assertEquals(returnedBook.getId(), 123L);
     }
 
-    @org.junit.Test
```

Comment:
Why is it being removed?

Reply:
Initially this test was introduced exactly for AT signature validation using JWKS but its more complex to configure and do not support key rotation